src/phrs.js

```diff
@@ -0,0 +1,7 @@
+function phrase() {
```

Comment:
This function would be more versatile if it could take an argument for the text, and you can set a default for that argument. Nice done!